src/main/java/emu/grasscutter/config/ConfigContainer.java

```diff
@@ -169,6 +171,8 @@ public static class Game {
         /* This is the port used in the default region. */
         public int accessPort = 0;
 
+        public boolean enableRandomEncryptSeed = true;
```

Comment:
```suggestion
        /* Enabling this will generate a unique packet encryption key for each player. */
        public boolean enableRandomEncryptSeed = true;
```